one/src/network.rs

```diff
@@ -110,12 +112,54 @@ impl<S> Ipfs<S> {
     pub fn api(&self) -> IpfsMetricsMiddleware<Arc<IpfsService<S>>> {
         self.api.clone()
     }
+    pub fn client(&self) -> IpfsClient {
+        IpfsClient(self.client.clone())
+    }
     pub async fn stop(self) -> Result<()> {
         self.p2p.stop().await?;
         Ok(())
     }
 }
 
+#[derive(Clone)]
+pub struct IpfsClient(P2pClient);
+
+#[async_trait]
+impl ceramic_api::P2PService for IpfsClient {
+    async fn peers(&self) -> Result<Vec<ceramic_api::PeerInfo>> {
+        self.0
+            .get_peers()
+            .await?
+            .into_iter()
+            .map(|(peer_id, addresses)| {
+                Ok(ceramic_api::PeerInfo {
+                    id: NodeId::try_from_peer_id(&peer_id)?,
+                    addresses,
+                })
+            })
+            .collect()
+    }
+    async fn peer_connect(&self, addrs: &[ceramic_api::Multiaddr]) -> Result<()> {
```

Comment:
Linking [the comment](https://github.com/ceramicnetwork/rust-ceramic/pull/626/files#r1873671829) from the other PR for visibility. Other than this concern, LGTM.